### PR TITLE
[JDK17 Release Tester] LPD-32260 Fix functional-bundle-tomcat-mysql57-jdk17_open and remove leftover jdk11 test batches in portal-release

### PR DESCRIPTION
**Pull request by julschong**
@vicnate5 @tinatian 
This is resubmitted from https://github.com/liferay-core-infra/liferay-portal/pull/7316#issuecomment-2270110428

These changes should be ok since they did change in job summary here: https://test-5-1.liferay.com/userContent/jobs/test-portal-release/builds/889/job-summary/index.html

I would like these to merge in soon so that release team can give me feedback on the the changes ive made so far.

### CODE REVIEW

**Comment by julschong** [comment]
ci:test:sf

File: test.properties [hunk 1]
```diff
@@ -7657,7 +7657,6 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
         functional-bundle-tomcat-db2115-jdk8,\
         functional-bundle-tomcat-mariadb104-jdk8,\
         functional-bundle-tomcat-mariadb106-jdk8,\
-        functional-bundle-tomcat-mysql57-jdk11_open,\
         functional-bundle-tomcat-oracle193-jdk8,\
```
Comment (julschong) [inline test.properties:7660]: remove this from dxp

Comment (julschong) [reply test.properties:7660]: and keep the jdk17_open test batch in ce

File: test.properties [hunk 2]
```diff
@@ -7882,11 +7881,7 @@ test.batch.run.property.query[functional-orcllinux9-tomcat90-mysql57-jdk8]=\
             (portal.acceptance != quarantine) AND \
             (portal.release == true) AND \
             (portal.upstream != quarantine)\
-        ) OR \
-        (portal.release ~ openjdk11)
```
Comment (julschong) [inline test.properties:7888]: this condition has these tests affected and need to notify search team.
image

**Comment by vicnate5** [comment]
@julschong @tinatian looks good to me

**Comment by julschong** [comment]
ci:forward

**Comment by julschong** [comment]
see https://github.com/brianchandotcom/liferay-portal/pull/153136

**Pull request closed without merging**